### PR TITLE
DocumentItem: show items on loan in 'where to find' section

### PR DESCRIPTION
**Pull request by Pineirin**
Screenshot from 2021-11-09 11-38-46

### CODE REVIEW

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js [hunk 1]
```diff
@@ -10,16 +10,29 @@ export default class DocumentItemBody extends Component {
...
+      ITEM_IN_TRANSIT_FOR_PICKUP: 'In Transit For Pickup',
+      ITEM_IN_TRANSIT_TO_HOUSE: 'In Transit To House',
+    };
 
-    if (onShelf) {
-      return <span className="success">On shelf</span>;
+    const isOnCirculation = this.itemStatus(item).isOnCirculation();
+    const onShelf = this.itemStatus(item).isOnShelf();
+
+    console.log(item);
```
Comment (NRodriguezcuellar) [inline src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js:29]: 👀

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js [hunk 2]
```diff
@@ -10,16 +10,28 @@ export default class DocumentItemBody extends Component {
...
   });
 
   statusLabel = (item) => {
-    const onShelf = this.itemStatus(item).IsOnShelf();
+    const circulationStates = {
+      ITEM_AT_DESK: 'At Desk',
+      ITEM_ON_LOAN: 'On Loan',
+      ITEM_IN_TRANSIT_FOR_PICKUP: 'In Transit For Pickup',
+      ITEM_IN_TRANSIT_TO_HOUSE: 'In Transit To House',
+    };
```
Comment (NRodriguezcuellar) [inline src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js:24]: For maintainability it is better to import objects from central files, i searched around the project and in the config/defaultConfig.js there is a records_config file with a property called "loanActiveStates" that looks pretty similar 😜

Comment (Pineirin) [reply src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js:24]: You are right, still, I cannot just use the defaultConfig, as I would need each element of the loanActiveStates to have a key (ej: ITEM_ON_LOAN) and a value (ej: 'On Loan'),  currently I have to manually add the value so I cannot take a lot of advantage of the config.

One possibility would be to had this object (circulationStates) to the config so it's easily reusable, but I don't even know if this is the solution that we should adopt (we could go for something simpler, like just putting 'On Loan' for everything).

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js [hunk 3]
```diff
@@ -10,16 +10,28 @@ export default class DocumentItemBody extends Component {
   }
 
   itemStatus = (item) => ({
-    IsOnShelf: () =>
-      invenioConfig.ITEMS.canCirculateStatuses.includes(item.status) &&
-      !item.circulation,
+    isOnCirculation: () =>
```
Comment (ntarocco) [inline src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js:13]: Let's keep the name we normally use:
```suggestion
    canCirculate: () =>
```

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js (hunk 2 above)
Comment (ntarocco) [reply src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js:24]: To make it simple, no need for the various circulationStates above, please just change it to:
```
if (canCirculate) {
     return onShelf ? (
        <span className="success">On shelf</span>
      ) : (
        <span>On loan</span>
      );
```
Should we give it a color when on loan ? orange maybe? ping @kprzerwa

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItems.js [hunk 4]
```diff
@@ -14,9 +14,8 @@ class DocumentItems extends Component {
   }
 
   itemStatus = (item) => ({
-    isOnShelf: () =>
-      invenioConfig.ITEMS.canCirculateStatuses.includes(item.status) &&
-      !item.circulation,
+    isOnCirculation: () =>
```
Comment (ntarocco) [inline src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItems.js:17]:
```suggestion
    canCirculate: () =>
```

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js (hunk 2 above)
Comment (Pineirin) [reply src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItemBody.js:24]: Had a small chat with @kprzerwa and @zzacharo. The final conclusion is to keep it grey.

**Review by kprzerwa — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]

**Pull request merged**